### PR TITLE
Fixes #24515 - add header tags (i.e title) to react components

### PR DESCRIPTION
**Pull request by amirfefer**
a mechanism for adding HTML tags into react pages.

### CODE REVIEW

File: webpack/containers/Application/Headers.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+
+const Header = props => (
+  <Helmet>
+    {Object.entries(props).map((h) => {
+      const TagName = h[0];
+
+      return <TagName key={TagName}>{h[1]} </TagName>;
```
Comment (sharvit) [inline webpack/containers/Application/Headers.js]: Could you use?
```js
Object.entries(props).map(([TagName, value]) =>
  <TagName key={TagName}>{value}</TagName>
);
```

Comment (amirfefer) [reply webpack/containers/Application/Headers.js]: thanks, looks better :+1:

**Comment by amirfefer** [comment]
Thanks @sharvit , could you have another quick look?

File: webpack/containers/Application/config.js [hunk 2]
```diff
@@ -18,15 +18,18 @@ export const links = [
...
     component: UpstreamSubscriptions,
   },
   {
+    text: 'Subscription Details',
     // eslint-disable-next-line no-useless-escape
     path: 'subscriptions/:id(\[0-9]*$\)',
     component: SubscriptionDetails,
   },
   {
+    text: 'Select Organization',
```
Comment (sjha4) [inline webpack/containers/Application/config.js]: The select organization page doesn't show the title `Select Organization`. It shows `RH Subscription` / `RH Repos` as the title. I believe that's not expected.
The other pages work great with the titles.. :+1:

File: webpack/containers/Application/config.js [hunk 3]
```diff
@@ -18,15 +18,18 @@ export const links = [
     component: WithOrganization(Subscriptions, '/subscriptions'),
   },
   {
+    text: 'Add Subscriptions',
```
Comment (tstrachota) [inline webpack/containers/Application/config.js]: I suppose the titles should be translated.

**Comment by amirfefer** [comment]
Titles are translated, plus `select organization` page's title is presented.

**Comment by sjha4** [comment]
Tested translations. Works as expected :+1: 
The organization select page title is RH Subscription/RH Repo..Below is the page I am looking at:
*****
rh sub

**Comment by amirfefer** [comment]
odd... it works for me:
screenshot from 2018-09-06 17-01-36

**Pull request merged**